calico_containers/calico_ctl/node.py

```diff
@@ -437,14 +443,28 @@ def _setup_ip_forwarding():
 
 
 def node_stop(force):
-    num_endpoints = len(client.get_endpoints(hostname=hostname,
-                                             orchestrator_id=DOCKER_ORCHESTRATOR_ID))
-    if num_endpoints and not force:
-        print "Current host has active endpoints so can't be stopped." \
-              " Force with --force"
-        return
+    """
+    Stop the Calico node.  This stops the containers (calico/node and
+    calico/node-libnetwork) that are started by calicoctl node.
+    """
+    endpoints = len(client.get_endpoints(hostname=hostname,
+                                         orchestrator_id=DOCKER_ORCHESTRATOR_ID))
```

Comment:
Yes - good point.  I agree - we should not limit this to docker orchestrator since the instructions are vague enough about workload tidyup.